tests/test_western/test_tremolo.py

```diff
@@ -0,0 +1,41 @@
+from neoscore.core.point import ORIGIN, ZERO
```

Comment:
ZERO should always be imported from neoscore.core.units. neoscore.core.point just happens to import ZERO, which in python unfortunately exposes it to re-exports. we should always import things from their original source.

Reply:
oops. sorry Ive done that before.